books/RayTracingTheRestOfYourLife.html

```diff
@@ -1102,31 +1128,42 @@
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
         }
         std::cout << std::fixed << std::setprecision(12);
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
         std::cout << "I = " << sum/N << '\n';
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
     }
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    [Listing [integ-xsq-2]: <kbd>[integrate_x_sq.cc]</kbd> Integrating $x^2$ with PDF]
-</div>
+    [Listing [integ-xsq-2]: <kbd>[integrate_x_sq.cc]</kbd> Explicit uniform PDF for $x^2$]
 
 
-Importance Sampling
---------------------
-Since we are sampling more where the PDF is big, we might expect less noise and thus faster
-convergence. In effect, we are steering our samples toward the parts of the distribution that are
-more _important_. This is why using a carefully chosen nonuniform PDF is usually called _importance
-sampling_.
+There are a couple of important things to emphasize. Every value of $x$ represents one sample of the
+function $x^2$ within the distribution [0,2]. We use a function $f$ to randomly select samples from
+within this distribution. We are no longer multiplying the output sum by $2$ to arrive at the final answer. We were previously multiplying the average over the interval (sum/N) times the length of the interval ($b - a = 2$). Here, we don't need to multiply by the interval length.
```

Comment:
“… within this distribution. ~We are no longer multiplying the output sum by $2$ to arrive at the final answer.~ We were previously multiplying the average over the interval (sum/N) times the length of the interval ($b - a = 2$). Here, we don't need to multiply by the interval length--that is, we no longer need to multiply the output sum by $2$ to arrive at the final answer.”